Loop/AppDelegate.swift

```diff
@@ -35,7 +36,10 @@ final class AppDelegate: UIResponder, UIApplicationDelegate {
         alertManager = AlertManager(rootViewController: rootViewController, expireAfter: Bundle.main.localCacheDuration ?? .days(1))
         deviceDataManager = DeviceDataManager(pluginManager: pluginManager, alertManager: alertManager)
         loopAlertsManager = LoopAlertsManager(alertManager: alertManager)
-
+        bluetoothStateManager = BluetoothStateManager()
```

Comment:
[nit] I would have expected that `BluetoothStateManager` would be injected into `LoopAlertsManager` and then `LoopAlertsManager` would add _itself_ as an observer, but that's a minor point.